morpheus-graphql-core/src/Data/Morpheus/Types/Error.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Data.Morpheus.Types.Error
```

Comment:
do we really need this ? .  i think having AST is sufficent

Reply:
Hi, I have removed this module. I think the downside is that this means users of the client library will need to import `Data.Morpheus.Types.Internal.AST` if they want to inspect the `GQLError` type (unless I am missing something). I think inspecting the error returned is pretty normal as you may want to branch based on what error you got.

I think ideally you should not need to import an `Internal` module for what I think is a common use case. Which isn't to say you need `Data.Morpheus.Types.Error`, you could also choose to export `GQLError (message, position)` from other existing modules.

Anyway, I'm not too fussed either way :)

Reply:
maybe later :) for now is ok how  it is.